From 11abda97baeb57109b20c99a1edad6f8205bedbc Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Sun, 17 Jun 2018 10:16:31 +0800
Subject: [PATCH 066/107] armv8: ls2080ardb: update copyright claim

This patch updates the copyright claim for the issues reported by Legal
Review.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
---
 board/freescale/ls2080ardb/ls2080ardb.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/board/freescale/ls2080ardb/ls2080ardb.c b/board/freescale/ls2080ardb/ls2080ardb.c
index a1311b2..cd5e9e4 100644
--- a/board/freescale/ls2080ardb/ls2080ardb.c
+++ b/board/freescale/ls2080ardb/ls2080ardb.c
@@ -1,6 +1,6 @@
 /*
- * Copyright (C) 2017 NXP Semiconductors
  * Copyright 2015 Freescale Semiconductor
+ * Copyright 2017 NXP
  *
  * SPDX-License-Identifier:	GPL-2.0+
  */
-- 
1.7.1

